packages/create-sei/templates/express-action-template/biome.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
i don't think we should include biome here, projects can add it themselves if they want it or other formatters